tests/integration/run_tests.rs

```diff
@@ -926,9 +926,7 @@ fn lock_redirects() {
   );
 }
 
-// TODO(2.0): this should be rewritten to a spec test and first run `deno install`
```

Comment:
I think it's fine to test this here and to only test with `nodeModulesDir: auto`.